src/components/Personalization/dom-actions/addNonceToInlineStyleElements.js

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { selectNodes } from "../../../utils/dom";
+import { SRC, STYLE } from "../../../constants/tagName";
+import { getAttribute, getNonce } from "./dom";
+
+const is = (element, tagName) => element.tagName === tagName;
```

Comment:
I'm not sure how much the rest of Alloy is compatible with XHTML files or how much we care tbh. If at.js does these types of things case-sensitively, then maybe our customers aren't using XHTML. 🤷‍♂️

Reply:
I just checked and we actually do convert tag names to uppercase in AppMeasurement. We have several instances in alloy where this would need to be updated if it turns out to be an issue, but given that you get uppercase (even if defined in lowercase) for HTML. I think I'd prefer not running a type conversion until it becomes a requirement.